Vagrantfile

```diff
@@ -2,31 +2,180 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
-
-  config.vm.define "keycloak", primary: true, autostart: true do |ood|
-    ood.vm.box = "centos/7"
-    ood.vm.network "forwarded_port", guest: 8080, host: 8080, auto_correct: true
-    ood.vm.provision "shell", inline: <<-SHELL
-      rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-7.noarch.rpm
-      yum -y install puppet-agent
-      source /etc/profile.d/puppet-agent.sh
-    SHELL
-    ood.vm.provision "shell", path: "vagrant-common.sh"
+
+  config.hostmanager.enabled = true
+  config.hostmanager.manage_host = true
+  config.hostmanager.manage_guest = true
+  config.hostmanager.ignore_private_ip = false
+  config.hostmanager.include_offline = false
+
+  config.vm.define "standalone", primary: true, autostart: true do |box|
+    box.vm.box = "centos/7"
+    box.vm.box_version = "1901.01"
+    box.vm.hostname = 'standalone.local'
+    box.vm.synced_folder ".", "/vagrant", type: "virtualbox"
+    box.hostmanager.manage_guest = true
+    box.hostmanager.aliases = %w(standalone)
+    box.vm.network "forwarded_port", guest: 8080, host: 8080, auto_correct: true
+    box.vm.provider 'virtualbox' do |vb|
+      vb.linked_clone = true
+      vb.gui = false
+      vb.memory = 1024
+      vb.customize ["modifyvm", :id, "--ioapic", "on"]
+      vb.customize ["modifyvm", :id, "--hpet", "on"]
+      vb.customize ["modifyvm", :id, "--audio", "none"]
+    end
+    box.vm.provision "shell" do |s|
+      s.path = "vagrant/install_agent.sh"
+    end
+    box.vm.provision "shell", path: "vagrant/vagrant-common.sh"
+  end
+
+  config.vm.define "standalone-ubuntu-1804", primary: false, autostart: false do |box|
+    box.vm.box = "ubuntu/bionic64"
+    box.vm.box_version = "20190903.0.0"
+    box.vm.hostname = 'standalone-ubuntu-1804.local'
+    box.vm.synced_folder ".", "/vagrant"
+    box.hostmanager.manage_guest = true
+    box.hostmanager.aliases = %w(standalone-ubuntu-1804)
+    box.vm.network "forwarded_port", guest: 8080, host: 8081, auto_correct: true
+    box.vm.provider 'virtualbox' do |vb|
+      vb.linked_clone = true
+      vb.gui = false
+      vb.memory = 1024
+      vb.customize ["modifyvm", :id, "--ioapic", "on"]
+      vb.customize ["modifyvm", :id, "--hpet", "on"]
+      vb.customize ["modifyvm", :id, "--audio", "none"]
+    end
+    box.vm.provision "shell" do |s|
+      s.path = "vagrant/install_agent.sh"
+    end
+    box.vm.provision "shell", path: "vagrant/vagrant-common.sh"
+  end
+
+  config.vm.define "db" do |box|
```

Comment:
Maybe name this "domain-db" to be in line with "standalone" and "standalone-ubuntu 18.04".?

Reply:
As an alternative to renaming all of this maybe just add a comment before each box to explain what their purpose is...